#Goal definition
bool confirmation
---
#Result definition
bool confirmationResult
---
#Feedback
